runtime/doc/lsp.txt

```diff
@@ -2135,4 +2257,130 @@ resolve_capabilities({server_capabilities})
                 Return: ~
                     (table) Normalized table of capabilities
 
+
+==============================================================================
+Lua module: vim.lsp.parser                                        *lsp-parser*
+
+parse({input})                                        *vim.lsp.parser.parse()*
```

Comment:
```suggestion
parse({input})                                        *vim.lsp.snippet.parse()*
```

Reply:
Right, I'll have to also change the paths in genvimdoc.py, I think that's where this is from